### PR TITLE
[#73] adapt package.json

### PR DESCRIPTION
**Pull request by ncedric**
Just adapting package.json to follow Angular package format v5.0

### CODE REVIEW

File: packages/stark-core/package.json [hunk 1]
```diff
@@ -1,6 +1,10 @@
 {
   "name": "@nationalbankbelgium/stark-core",
   "version": "0.0.0-PLACEHOLDER-VERSION",
+  "module": "./esm5/stark-core.js",
+  "es2015": "./es2015/stark-core.js",
+  "main": "bundles/stark-core.umd.js",
```
Comment (SuperITMan) [inline packages/stark-core/package.json]: Why is it no "main": "./bundles/stark-core.umd.js" as it is for "module" and "es2015" ?

File: packages/stark-core/package.json [hunk 2]
```diff
@@ -1,6 +1,10 @@
...
+  "typings":"stark-core.d.ts",
```
Comment (SuperITMan) [inline packages/stark-core/package.json]: Maybe "types": "stark-core.d.ts" could be better than "typings" ?

Comment (christophercr) [reply packages/stark-core/package.json]: Indeed, "typings" was the previous name of this property, now it should be "types" (although both work fine)

File: packages/stark-core/package.json (hunk 1 above)
Comment (ncedric) [reply packages/stark-core/package.json]: @SuperITMan 
I have implemened by following the documentation : 
https://docs.google.com/document/d/1CZC2rcpxffTDfRDs6p1cfbmKNLA6x5O-NtkJglDaBVs/preview

Kr,
Cédric

**Pull request merged**